test.sh

```diff
@@ -58,6 +58,23 @@ cd $REPO_ROOT_DIR
 
 # Get a CCM cluster if not already configured (see available settings in dcos-commons/tools/README.md):
 if [ -z "$CLUSTER_URL" ]; then
+    if [ "$SECURITY" = "strict" ]; then
```

Comment:
Strict mode tests fail in launch_ccm_cluster.py's use of create_service_account.sh which depends on the dcos binary.